src/shared/api/cbioportalClientInstance.ts

```diff
@@ -2,4 +2,22 @@ import { CBioPortalAPI } from 'cbioportal-ts-api-client';
 
 const client = new CBioPortalAPI();
 
+import { isClickhouseMode } from 'config/config';
+import { proxyColumnStore } from 'shared/api/proxyColumnStore';
+import { overrideApiRequestForColumnStore } from 'shared/api/overrideApiRequestForColumnStore';
+
+const clientColumnStore = new CBioPortalAPI();
+
+export function getClient() {
+    return isClickhouseMode() ? clientColumnStore : client;
+}
+
+overrideApiRequestForColumnStore(clientColumnStore);
+
+proxyColumnStore(clientColumnStore, 'fetchSamples');

```

Comment:
@fuzhaoyuan when you fire up the branch, you add an entry here corresponding to the client method (which corresponds to your endpoint) and then it will call the clickhouse-ified version